doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -104,10 +104,23 @@ The following rules govern create, read, update, and delete (CRUD) level access.
 ### 6.3.2 System Hardening
 All infrastructure should undergo system hardening, establishes processes to govern the hardening, and documents to cover at a minimal for the following areas:
 
-#### 6.3.2.1 Function
+#### 6.3.2.1 Password policy
+For all infrastructure components, passwords must be hardened and a strict password policy must be applied (req.sec.gen.002).
+
+Passwords must be strengthen:
+-	All vendors default passwords must be changed
+-	Passwords must contain at least 8 characters
+-	Passwords must contain at least one upper case letter, one lower case letter and one non-alphabetic character
```

Comment:
should we have: 
```suggestion
-	Passwords must contain at least one upper case letter, one lower case letter, one numeral and one special (non-alphanumeric) character.
```

Reply:
@pgoyal01  In my view, it was the minimal requirement for passwords for all kind of accounts. For administration, the password can be strenghtened as you proposed.
Can it be fine to add a difference between users with or without administration privileges?

Reply:
@karinesevilla Yes -- that would be better

Reply:
@goyal Done, 'Administration privileges accounts'  added